### PR TITLE
Fixed Broken Xamarin.com URL references

### PR DESCRIPTION
**Pull request by saamerm**
* Fixed all remaining broken URL references to Xamarin.com
* Fixed the URL in a lot of places where users looking for Xamarin Samples are taken to the right place
* Ensured that all the Deep Linking URL was correct and that the Title of the article matched the text inside the `<a href>` tag
* Fixed the behaviors URLs to take users to the right place
* Fixed the Intro to Data Binding link to go to the blog
* Fixed the link for the Xamarin Forms Images reference
* Fixed the Image source in the examples to take display a Xamarin PNG that actually works, since all other pngs were of different formats that may not have worked
* Corrected the webview example, so that users dont get an error on android when they are trying to go to the older blog.xamarin.com

**Testing**
----------
* Ensured that these were actual issues by going to the docs page for behaviors and seeing that it in fact have the wrong URL and took users to the wrong place

### CODE REVIEW

**Comment by saamerm** [comment]
@Redth looks good to me. @samhouts, any comments?

**Review by samhouts — APPROVED** [review]

File: docs/Xamarin.Forms/Behavior.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
   <Docs>
     <summary>Base class for generalized user-defined behaviors that can respond to arbitrary conditions and events.</summary>
     <remarks>Application developers should specialize the <see cref="T:Xamarin.Forms.Behavior`1" /> generic class, instead of directly using <see cref="T:Xamarin.Forms.Behavior" />.</remarks>
-    <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/application-fundamentals/behaviors/creating/">Xamarin.Forms Behaviors</related>
+    <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/behaviors/">Xamarin.Forms Behaviors</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/Behavior.xml:22]:
```suggestion
    <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/behaviors/">Xamarin.Forms Behaviors</related>
```

File: docs/Xamarin.Forms/Behavior`1.xml [hunk 2]
```diff
@@ -26,7 +26,7 @@
     <typeparam name="T">The type of the objects with which this <see cref="T:Xamarin.Forms.Behavior`1" /> can be associated.</typeparam>
     <summary>Base generic class for generalized user-defined behaviors that can respond to arbitrary conditions and events.</summary>
     <remarks>Application developers should specialize this generic class, instead of directly using <see cref="T:Xamarin.Forms.Behavior" />.</remarks>
-    <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/application-fundamentals/behaviors/creating/">Xamarin.Forms Behaviors</related>
+    <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/behaviors/">Xamarin.Forms Behaviors</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/Behavior`1.xml:29]:
```suggestion
    <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/behaviors/">Xamarin.Forms Behaviors</related>
```

File: docs/Xamarin.Forms/Button.xml [hunk 3]
```diff
@@ -69,7 +69,7 @@
         BorderColor="Black"
         HorizontalOptions="Center"/>
 ]]></code>
-        <block type="note">To display the <c>icon.png</c> image with the above XAML, the developer must place it in the appropriate location, with the appropriate build action, specific to each platform project. The <format type="text/html"><a href="https://developer.xamarin.com/guides/xamarin-forms/user-interface/images/">Images</a></format> topic explains this and other important details about working with images in Xamarin.Forms.</block>
+        <block type="note">To display the <c>icon.png</c> image with the above XAML, the developer must place it in the appropriate location, with the appropriate build action, specific to each platform project. The <format type="text/html"><a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/images">Images</a></format> topic explains this and other important details about working with images in Xamarin.Forms.</block>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/Button.xml:72]:
```suggestion
        <block type="note">To display the <c>icon.png</c> image with the above XAML, the developer must place it in the appropriate location, with the appropriate build action, specific to each platform project. The <format type="text/html"><a href="https://docs.microsoft.com/xamarin/xamarin-forms/user-interface/images">Images</a></format> topic explains this and other important details about working with images in Xamarin.Forms.</block>
```

File: docs/Xamarin.Forms/Entry.xml [hunk 4]
```diff
@@ -56,7 +56,7 @@ View CreateLoginForm ()
   };
 }
             ]]></code>
-        <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://developer.xamarin.com/samples/tag/Xamarin.Forms/" target="_blank">Sample Applications</a></format> page, has an EntryDemoPage.cs file. This file contains a more complete example.</para>
+        <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/samples/" target="_blank">Sample Applications</a></format> page, has an EntryDemoPage.cs file. This file contains a more complete example.</para>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/Entry.xml:59]: These links point to both the specific example, and the more general page with all the XF samples :)
```suggestion
        <para>The <format type="text/html"><a href="https://docs.microsoft.com/samples/xamarin/xamarin-forms-samples/formsgallery/" target="_blank">FormsGallery sample</a></format>, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/samples/browse/?products=xamarin&term=Xamarin.Forms&terms=Xamarin.Forms" target="_blank">Sample Applications</a></format> page, has an EntryDemoPage.cs file. This file contains a more complete example.</para>
```

File: docs/Xamarin.Forms/IAppLinkEntry.xml [hunk 5]
```diff
@@ -12,7 +12,7 @@
   <Docs>
     <summary>Interface that represents a deep application link in an app link search index.</summary>
     <remarks>To be added.</remarks>
-    <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+    <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinkEntry.xml:15]:
```suggestion
    <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinkEntry.xml [hunk 6]
```diff
@@ -33,7 +33,7 @@
         <summary>Gets or sets an application-specific URI that uniquely describes content within an app.</summary>
         <value>An application-specific URI that uniquely describes content within an app.</value>
         <remarks>To be added.</remarks>
-        <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+        <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinkEntry.xml:36]:
```suggestion
        <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinkEntry.xml [hunk 7]
```diff
@@ -54,7 +54,7 @@
         <summary>Gets or sets a description that appears with the item in search results.</summary>
         <value>To be added.</value>
         <remarks>To be added.</remarks>
-        <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+        <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinkEntry.xml:57]:
```suggestion
        <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinkEntry.xml [hunk 8]
```diff
@@ -75,7 +75,7 @@
         <summary>Gets or sets a value that tells whether the item that is identified by the link entry is currently open.</summary>
         <value>A value that tells whether the item that is identified by the link entry is currently open.</value>
         <remarks>Application developers can set this value in <see cref="E:Xamarin.Forms.Application.PageAppearing" /> and <see cref="E:Xamarin.Forms.Application.PageDisappearing" /> methods to control whether the app link is shown for indexing or Handoff.</remarks>
-        <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+        <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinkEntry.xml:78]:
```suggestion
        <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinkEntry.xml [hunk 9]
```diff
@@ -96,7 +96,7 @@
         <summary>Gets a dictionary of application-specific key-value pairs.</summary>
         <value>A dictionary of standard and application-specific key-value pairs that is used for Handoff on the iOS platform.</value>
         <remarks>The standard keys are <c>contentType</c>, <c>associatedWebPage</c>, and <c>shouldAddToPublicIndex</c>.</remarks>
-        <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+        <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinkEntry.xml:99]:
```suggestion
        <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinkEntry.xml [hunk 10]
```diff
@@ -117,7 +117,7 @@
         <summary>Gets or sets a small image that appears with the item in search results.</summary>
         <value>A small image that appears with the item in search results</value>
         <remarks>To be added.</remarks>
-        <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+        <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinkEntry.xml:120]:
```suggestion
        <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinkEntry.xml [hunk 11]
```diff
@@ -138,7 +138,7 @@
         <summary>Gets or sets the title of the item.</summary>
         <value>The title of the item.</value>
         <remarks>To be added.</remarks>
-        <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+        <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinkEntry.xml:141]:
```suggestion
        <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinks.xml [hunk 12]
```diff
@@ -12,7 +12,7 @@
   <Docs>
     <summary>Interface that defines the methods that are required for registering and deregistering links to content in apps.</summary>
     <remarks>Developers should note that extra steps are required to enable deep linking on the Android platform. See the related article that is linked below.</remarks>
-    <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+    <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinks.xml:15]:
```suggestion
    <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinks.xml [hunk 13]
```diff
@@ -36,7 +36,7 @@
         <param name="appLinkUri">The <see cref="T:System.Uri" /> that represents the link to remove from the app index.</param>
         <summary>Removes the provided application link from the application index.</summary>
         <remarks>This method has no effect on the Android platform.</remarks>
-        <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+        <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinks.xml:39]:
```suggestion
        <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinks.xml [hunk 14]
```diff
@@ -60,7 +60,7 @@
         <param name="appLink">The <see cref="T:Xamarin.Forms.IAppLinkEntry" /> to remove from the app index.</param>
         <summary>Removes the provided application link from the application index.</summary>
         <remarks>This method has no effect on the Android platform.</remarks>
-        <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+        <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinks.xml:63]:
```suggestion
        <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/IAppLinks.xml [hunk 15]
```diff
@@ -84,7 +84,7 @@
         <param name="appLink">The <see cref="T:Xamarin.Forms.IAppLinkEntry" /> to add to the app index.</param>
         <summary>Adds the provided application link to the application index.</summary>
         <remarks>To be added.</remarks>
-        <related type="article" href="https://developer.xamarin.com/guides/xamarin-forms/working-with/deep-linking/">Application Indexing and Deep Linking</related>
+        <related type="article" href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/IAppLinks.xml:87]:
```suggestion
        <related type="article" href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/deep-linking">Application Indexing and Deep Linking</related>
```

File: docs/Xamarin.Forms/Image.xml [hunk 16]
```diff
@@ -53,7 +53,7 @@
         <code lang="csharp lang-csharp"><![CDATA[
 var image = new Image { Source = "picture.png" };]]></code>
       </example>
-      <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://developer.xamarin.com/samples/tag/Xamarin.Forms/" target="_blank">Sample Applications</a></format> page, has a ImageDemoPage.cs file. This file contains a longer and more complete example.</para>
+      <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/samples/" target="_blank">Sample Applications</a></format> page, has a ImageDemoPage.cs file. This file contains a longer and more complete example.</para>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/Image.xml:56]: As before - these links point to both the specific example, and the more general page with all the XF samples :)
```suggestion
      <para>The <format type="text/html"><a href="https://docs.microsoft.com/samples/xamarin/xamarin-forms-samples/formsgallery/" target="_blank">FormsGallery sample</a></format>, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/samples/browse/?products=xamarin&term=Xamarin.Forms&terms=Xamarin.Forms" target="_blank">Sample Applications</a></format> page, has an ImageDemoPage.cs file. This file contains a longer and more complete example.</para>
```

File: docs/Xamarin.Forms/Label.xml [hunk 17]
```diff
@@ -60,7 +60,7 @@ public class App : Application
     };
 }
 ]]></code>
-        <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://developer.xamarin.com/samples/tag/Xamarin.Forms/" target="_blank">Sample Applications</a></format> page, has a LabelDemoPage.cs file. This file contains a longer and more complete example.</para>
+        <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/samples/" target="_blank">Sample Applications</a></format> page, has a LabelDemoPage.cs file. This file contains a longer and more complete example.</para>
```
Comment (rachelkang) [inline docs/Xamarin.Forms/Label.xml:63]: As before - these links point to both the specific example, and the more general page with all the XF samples :)
```suggestion
        <para>The <format type="text/html"><a href="https://docs.microsoft.com/samples/xamarin/xamarin-forms-samples/formsgallery/" target="_blank">FormsGallery sample</a></format>, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/samples/browse/?products=xamarin&term=Xamarin.Forms&terms=Xamarin.Forms" target="_blank">Sample Applications</a></format> page, has a LabelDemoPage.cs file. This file contains a longer and more complete example.</para>
```

**Review by rachelkang — CHANGES_REQUESTED** [review]
See suggestions/comments - remove locales from all URLs and change URL for sample apps to All Samples page https://docs.microsoft.com/en-us/samples/browse/?products=xamarin&term=Xamarin.Forms&terms=Xamarin.Forms

**Comment by saamerm** [comment]
@rachelkang good suggestions. Made the changes

**Comment by saamerm** [comment]
@samhouts the errors in question seem to be from urls that work fine. What can we do?

File: docs/Xamarin.Forms/ImageCell.xml [hunk 18]
```diff
@@ -52,7 +52,7 @@ namespace FormsGallery
                         {
                             // Some differences with loading images in initial release.
                             ImageSource =
-                                Device.OnPlatform(ImageSource.FromUri(new Uri("http://xamarin.com/images/index/ide-xamarin-studio.png")),
+                                Device.OnPlatform(ImageSource.FromUri(new Uri("https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/xamarin/xamarin.png")),
```
Comment (rachelkang) [inline docs/Xamarin.Forms/ImageCell.xml:55]: Would you also swap out the image url to this one from our dotnet swag repo? Thanks :)
https://github.com/dotnet-foundation/swag/tree/master/xamarin

**Comment by davidbritch** [comment]
[Microsoft support for Xamarin](https://dotnet.microsoft.com/en-us/platform/support/policy/xamarin) will end on May 1, 2024 for all Xamarin SDKs including Xamarin.Forms. In preparation for this, all issues and PRs in this repository are being closed.

.NET MAUI is the replacement for Xamarin.Forms, and its API docs are built from /// in the [dotnet/maui](https://github.com/dotnet/maui) repository. Therefore, any PRs for the .NET MAUI API docs should be opened in that repository.

**Pull request closed without merging**